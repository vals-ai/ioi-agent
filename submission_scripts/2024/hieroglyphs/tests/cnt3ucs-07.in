version https://git-lfs.github.com/spec/v1
oid sha256:74b5b68c0290c32041df7318872f2398f137f8df20ba7389f2e63888fa23a94b
size 1289197
