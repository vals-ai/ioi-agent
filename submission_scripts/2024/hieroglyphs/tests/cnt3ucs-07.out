version https://git-lfs.github.com/spec/v1
oid sha256:8698c33792fc9e0ffac2c34f9ccf91334012f5256a9a84599a93d385a8b052e1
size 268725
